plugins/modules/ecs_taskdefinition.py

```diff
@@ -766,7 +766,7 @@ def main():
         launch_type=dict(required=False, choices=['EC2', 'FARGATE']),
         cpu=dict(),
         memory=dict(required=False, type='str'),
-        ephemeral_storage=dict(required=False,type='int')
+        ephemeral_storage=dict(required=False, default=20, type='int')
```

Comment:
as discussed EC2 deploy does not like this parameter so I think we need to take it out for ec2 service deploy